tests/tests_fabric/strategies/test_fsdp.py

```diff
@@ -169,6 +169,32 @@ def __init__(self):
         ckpt_mock.assert_called_with(fsdp_mock(), checkpoint_wrapper_fn=ANY, check_fn=ANY)
 
 
+@RunIf(min_torch="1.13", min_cuda_gpus=1)
+def test_fsdp_manual_activation_checkpointing():
```

Comment:
Since it is using Fabric end-to-end, could you move it to test_fsdp_integration.py? Or alternatively, you could write a unit test using just the FSDPStrategy (calling strategy.setup_module).

Reply:
Moved the file. I couldn't get it to work by just using FSDPStrategy without having to write the launching logic in the test. I don't want to fully mock FSDP in case FSDP changes it activation checkpointing in the future so that it doesn't reapply it